composer.json

```diff
@@ -25,7 +25,7 @@
         "php": "^7.1 || ^8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
+        "sminnee/phpunit": "^5.7",
```

Comment:
Was this specifically required for this PR, or just something that's not merged upstream yet as parallel work?

Reply:
Will try removing it, but I was getting a fatal without it.. some kind of incompatibility in Sebastian\Comparator with PHPUnit. I noticed a lot of other core modules are using Sam's fork.

Reply:
Yeah no worries, `recipe-cms` uses it as well.